src/Psalm/Config.php

```diff
@@ -1292,12 +1276,11 @@ private static function requirePath(string $path) : void
     /**
      * @template T
      *
-     * @param  string $path
      * @param  T::class $must_extend
      *
      * @return class-string<T>
      */
-    private function getPluginClassForPath(Codebase $codebase, $path, $must_extend)
+    private function getPluginClassForPath(Codebase $codebase, string $path, $must_extend)
```

Comment:
`string` type can be added to `$must_extend` while keeping `T::class` in the docblock to verify that it is a class name.